content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -444,6 +444,8 @@ either be a snapshot file from a previous backup operation, or from a remaining
    ```shell
    export ETCDCTL_API=3
    etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
+or 
```

Comment:
This line breaks the code snippet formatting, and isn't valid within a shell script.